frickfeed/templates/base.html

```diff
@@ -16,30 +49,26 @@
     <script src="static/js/lib/leaflet.js"></script>
     <style>
       .shippingRecord {
-        height: 180px;
+        height: 250px;
       }
     </style>
-    <!-- End Leaflet for Image Viewer -->
   </head>
 
   <body>
     <!-- Header -->
     <header>
-      <paper-toolbar>
+      <paper-toolbar paper-toolbar-color="#000">
+        <paper-icon-button icon="menu" style="color: white;"></paper-icon-button>
```

Comment:
again, are these styles here for performance reasons? also, cool polymer element!!